puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/plugins/foreman-tasks.yaml

```diff
@@ -4,5 +4,9 @@
     branch:
       - master:
           foreman_branch: develop
+      - 0.15.x:
+          foreman_branch: 1.22-stable
+      - 0.16.x:
```

Comment:
Nit: I think it's more logical to sort newest -> oldest or oldest -> newest but this feels a bit inconsistent.